laiaaa0/assignment3/Dictionary.h

```diff
@@ -1,22 +1,27 @@
 #include <vector>
+#include <string>
 #include <iostream>
 #include <map>
+#include <set>
+
+#ifndef DICTIONARY_H
+#define DICTIONARY_H
+
 class Dictionary{
-  private:
 
+  private:
     std::map<std::string, bool> dictionary_words_;
```

Comment:
Shouldn't this actually be a `std::set<std::string>` ?
I you didn't not had RemoveWord dictionary_prefixes_ could also be a set.